src/navigation/underline-nav.scss

```
@@ -23,16 +23,14 @@
   border: 0;
   // stylelint-disable-next-line primer/borders
   border-bottom: 2px $border-style rgba($border-gray-dark, 0);
-  transition: border-bottom-color 0.36s ease-in;
 
   &:hover,
   &:focus {
     text-decoration: none;
     border-bottom-color: $border-gray-dark;
     outline: 1px dotted transparent; // Support Firfox custom colors
     outline-offset: -1px;
-    transition-timing-function: ease-out;
-    transition-duration: 0.12s;
+    transition: border-bottom-color 0.36s ease-in;
```

Maybe we could keep the shorter transition timing to still make it feel "snappy" on hover?

```suggestion
    transition: border-bottom-color 0.12s ease-out;
```